src/components/Proposal/View/AboutBidProcess.tsx

```diff
@@ -24,12 +24,12 @@ const getTenderConfig = () => {
 }
 
 const getOpenForBidsConfig = (hasBid: boolean, hasBidProposals: boolean) => {
-  if (hasBid) {
-    return { status: ProcessStatus.Pending, statusText: 'page.proposal_bidding_tendering.open_for_bids_begins' }
+  if (hasBidProposals) {
+    return { status: ProcessStatus.Passed, statusText: 'page.proposal_bidding_tendering.open_for_bids_closed' }
   }
 
-  if (hasBidProposals) {
-    return { status: ProcessStatus.Passed, statusText: 'page.proposal_bidding_tendering.open_for_bids_inbound' }
+  if (hasBid) {
```

Comment:
the difference between `hasBid` and `hasBidProposals` is whether the bids are published or not? In that case I might rename `hasBid` to `hasUnpublishedBid`